src/HookListener/Registration.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace Terminal42\AutoRegistrationBundle\HookListener;
```

Comment:
We always use `EventListener` namespace, hooks are just events with a different markup.